### ISSUE TITLE
Russian

### ISSUE THREAD

**Issue by vicefi** [hudsonfoo/jQuery-Bacon#4]
This script don't work in Russian letters

**Comment by vcalaelen** [comment]
Yes, I can too confirm that the plugin doesn't work with Cyrillic(Bulgarian, Russian, etc.).

**Comment by MrStepson** [comment]
The issue is related to regular expression **/\W\b/** that is used to find beginings of  words. This expression doesn't work with non-Latin characters. Try to change it to **/[^A-Za-z0-9_\u0410-\u044f][A-Za-z0-9_\u0410-\u044f]+/**. Also please note that your html-file with cyrillic text should be saved as Unicode.
